src/platform/intel/cavs/platform.c

```diff
@@ -11,13 +11,10 @@
 #if (CONFIG_CAVS_LPS)
```

Comment:
will CONFIG_CAVS_LPS be set in one platform?

Reply:
@btian1 I'll follow-up with separate PR to remove CAVS_LPS stuff. This needs a bit more surgery so I'll do that in a separate patch.